Dockerfile

```diff
@@ -11,15 +11,6 @@ ARG CA_PATH_VALUE
 ARG PROMETHEUS_BASIC_AUTH
 RUN make build_encrypted
 
-FROM alpine:3.15.2 as advanced
```

Comment:
I think you need to add this back too if you bring the action back :)

Reply:
definitely yes